content/guides/spring/spring-route-authorization-testing.md

```diff
@@ -0,0 +1,550 @@
+---
+draft: false
+date: 2021-05-05T10:40:17-06:00
+tags: ["spring","testing", "spring security"]
+title: "Spring Route Authorization Testing"
+description: >
+    Dynamic testing of your Spring Security route authorizations
+date: "2021-05-12"
+topics:
+- Spring
+- Testing
+tags:
+- Spring
+- Spring Security
+- Spring Testing
+# Author(s)
+team: 
+- Toby Rumans
+---
+
+Spring Security enables us to lock down routes in applications to particular roles. 
+
+Testing route authorizations are often manual processes, that are prone to human error and often have complicated setup. 
+
+The following are dynamic tests that ensure your route authorization configuration is up-to-date, and has 100 percent coverage.
+
+**Code examples** 
+
+<https://github.com/tobocop/spring-route-authorization-testing>
+
+_Note_: The solutions provided were coded for `spring-boot-starter-security:2.4.4` and junit 5.
```

Comment:
The `_Note:_` prefix is fine here, but we do have the ability to put this in a "callout" box with a warning or info symbol, if you'd like.

Reply:
done